### ISSUE TITLE
Repo IntelliSense weirdness

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode-github-issue-notebooks#39]
First, I noticed that when I type "repo" and hit enter on intellisense, the suggestions doesn't come up because there is no `:` so i have to manually add it. it then takes a few seconds for the suggestions to come up, but as I type to filter the list it's not update properly and at some point it goes away after I am done and comes back seconds later. Lots of weirdness, not sure if this is related to recent SSO sign on for GH.

![Kapture 2020-06-04 at 10 09 08](https://user-images.githubusercontent.com/35271042/83789418-8c9bcc00-a64b-11ea-9390-c8060e92c67d.gif)

**Comment by jrieken** [comment]
There is little that I can do. Fetching repos from GH is surprisingly hard and slow so that these kind of completions are fetch as you type, e.g whenever you type a longer prefix we use that prefix to query GH. That takes its time...

**Comment by misolori** [comment]
Not sure if showing the `Loading...` item while we fetch would help here because when there is no indication it feels like it's broken.

**Comment by jrieken** [comment]
Yeah, I actually don't know why we don't show it. It must be because IntelliSense was triggered implicitly, e.g by just typing a word, and not via `ctrl+space`.
